tests/test_gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,21 @@
+from os.path import exists
+from src.gauss_pdf_and_cdf import (GaussianProbabilisticSetting,
+                                   BetaProbabilisticSetting)
+import cv2
+
+
+def test_figure(distribution_type="gaussian"):
```

Comment:
create a function to call this function (maybe you should rename) twice with both gaussian and beta